Clarify Forward=0 does not affect SUBSCRIBE_DONE (#947)
moq-wg/moq-transport: draft-ietf-moq-transport.md

past are retrieved using FETCH ({{message-fetch}}).

A Subscription can also request a publisher to not forward Objects for a given

> track by setting the `Forward` field to 0. The SUBCRIBE_DONE

Reviewer: nit: I'd probably stick this at the end of the paragraph, since it seems sort of obvious.  I mean, Group Order doesn't change SUBSCRIBE_DONE either

> track by setting the `Forward` field to 0. The SUBCRIBE_DONE
> ({{message-subscribe-done}}) control message is not affected by this flag.

Reviewer: ```suggestion
track by setting the `Forward` field to 0. Control messages, such as SUBCRIBE_DONE
({{message-subscribe-done}}) are still sent.
```
